src/WhatsAppCloudApi.php

```diff
@@ -13,7 +13,7 @@ class WhatsAppCloudApi
     /**
      * @const string Default Graph API version.
      */
-    public const DEFAULT_GRAPH_VERSION = 'v15.0';
+    public const DEFAULT_GRAPH_VERSION = 'v17.0';
```

Comment:
Could this break the code? https://developers.facebook.com/docs/whatsapp/business-platform/changelog#february-2--2023

Reply:
Nope, the template categories were resolved by an issue opened sometime back: https://github.com/netflie/whatsapp-cloud-api/issues/110 . All those categories were made available from June 1, 2023